examples/features/authentication/server/main.go

```diff
@@ -72,18 +73,20 @@ func main() {
 	}
 }
 
-type ecServer struct{}
+type ecServer struct {
+	pb.UnimplementedEchoServer
+}
 
-func (s *ecServer) UnaryEcho(ctx context.Context, req *ecpb.EchoRequest) (*ecpb.EchoResponse, error) {
-	return &ecpb.EchoResponse{Message: req.Message}, nil
+func (s *ecServer) UnaryEcho(ctx context.Context, req *pb.EchoRequest) (*pb.EchoResponse, error) {
+	return &pb.EchoResponse{Message: req.Message}, nil
 }
-func (s *ecServer) ServerStreamingEcho(*ecpb.EchoRequest, ecpb.Echo_ServerStreamingEchoServer) error {
+func (s *ecServer) ServerStreamingEcho(*pb.EchoRequest, pb.Echo_ServerStreamingEchoServer) error {
 	return status.Errorf(codes.Unimplemented, "not implemented")
 }
-func (s *ecServer) ClientStreamingEcho(ecpb.Echo_ClientStreamingEchoServer) error {
+func (s *ecServer) ClientStreamingEcho(pb.Echo_ClientStreamingEchoServer) error {
 	return status.Errorf(codes.Unimplemented, "not implemented")
 }
-func (s *ecServer) BidirectionalStreamingEcho(ecpb.Echo_BidirectionalStreamingEchoServer) error {
+func (s *ecServer) BidirectionalStreamingEcho(pb.Echo_BidirectionalStreamingEchoServer) error {
 	return status.Errorf(codes.Unimplemented, "not implemented")
 }
```

Comment:
Why do we need to define these methods which are unimplemented? Won't we get it from the `UnimplementedEchoServer` that we have embedded?

Reply:
We don't.  I made a mechanical change across many files and didn't check for these.  I'll take another pass.

Reply:
Done!